### ISSUE TITLE
Plots overlap with vstack, hstack, gridstack

### ISSUE THREAD

**Issue by cmcaine** [GiovineItalia/Gadfly.jl#1299]
MCE:

```julia
using Gadfly

# Looks fine.
plot(x = 1:10, y = rand(10) .* (1:10), Geom.point)
# Plots overlap.
vstack((plot(x = 1:10, y = rand(10) .* (1:10), Geom.point) for _=1:10)...)
```

![foo](https://user-images.githubusercontent.com/6000761/61400680-b085e000-a8c8-11e9-8afb-103edac7292c.png)

**Comment by Mattriks** [comment]
I think you are asking that plots should know how big they are (rather than plotting at a default size)
 #1058, GiovineItalia/Compose.jl#28

**Comment by cmcaine** [comment]
Yes, I suspect that's the root issue. vstack could also tell compose to plot them bigger.

In the meantime a tedious workaround is to use e.g. `set_default_plot_size(500px, 700px)`. In Jupyter I think you can pipe to `SVG(500px, 700px)`, but not from the command line repl.
